pkg/ast/ast_operation_definition.go

```diff
@@ -80,6 +80,18 @@ func (d *Document) AddImportedVariableDefinitionToOperationDefinition(operationD
 		append(d.OperationDefinitions[operationDefinition].VariableDefinitions.Refs, variableDefinition)
 }
 
+func (d *Document) OperationNameExists(operationName string) bool {
+	nameExists := false
+	for i := 0; i < len(d.OperationDefinitions); i++ {
+		if d.OperationDefinitionNameString(i) == operationName {
+			nameExists = true
```

Comment:
maybe just `return true` here and `return false` at the end?

Reply:
thats better! I copied the code over and wasn't aware of this better solution in this case :+1: